tests/specs/controllers/OpenEAppController.test.js

```diff
@@ -88,7 +88,9 @@ describe('OpenEAppController', () => {
             _sails: {
                 config: {
                     hmacKey: '123',
-                    customURLs: '123',
+                    customURLs: {
+                        userServiceURL: 'http://localhost/3000/'
```

Comment:
Is this supposed to be `localhost:3000`?

Reply:
Yeah this could be anything really, it's just a string for testing